lib/newFileMenu.ts

```diff
@@ -23,10 +23,32 @@
 import type { Folder, Node } from './index'
 import logger from './utils/logger'
 
+export enum NewMenuEntryCategory {
+	/**
+	 * For actions where the user is intended to upload from their device
+	 */
+	UploadFromDevice = 0,
+	/**
+	 * For actions that create new nodes on the server without uploading
+	 */
+	CreateNew = 1,
+	/**
+	 * For everything not matching the other categories
+	 */
+	Other = 2,
+}
+
 export interface Entry {
 	/** Unique ID */
 	id: string
 
+	/**
+	 * Category to put this entry in
+	 * Added with v3.3.0 (supported since Nextcloud 30)
```

Comment:
Maybe https://jsdoc.app/tags-since ?

Reply:
I tried but it results in eslint warning :(

Reply:
Found a solution. Not that nice, but better :white_check_mark: